### DISCUSSION TITLE
day2  turtlesim 관련 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by ohzzang** [Road-Balance/gcamp_ros2_basic#30] [Q&A] [answered]
![ddd](https://user-images.githubusercontent.com/96938924/178447804-c04d3960-28e3-43fa-9258-9a13d174ba35.jpg)
알려주신대로 Xlaunch 설정을 맞게하고 재부팅도 하고 해도 결과가 계속 똑같이 나옵니다.

**Comment by kimsooyoung** [comment, marked as answer]
1. 이전 1일차 강의에 3줄 export 부분을 찾아서 실행해보세요
2. Gazebo는 잘 나오나요?
3. 혹시 듀얼모니터를 사용하고 계신가요?

**Reply by ohzzang** [reply]
Gazebo는 잘나옵니다! 듀얼모니터는 사용하지 않고 있습니다.

**Reply by ohzzang** [reply]
![image](https://user-images.githubusercontent.com/96938924/178496938-86f6354b-e548-4c73-84a0-b5b72bb16a9a.png)

![image](https://user-images.githubusercontent.com/96938924/178496869-24f5424a-75d6-41ee-bde3-3446fe831523.png)
추가로  만들어주신 패키지를 빌드하고  launch 하였을 때 gazebo가 실행되지 않는 이유가 있을까요?? 터미널에서 gazebo 만 실행시키면 되는 것은 확인하였습니다. 이후 종료하고 사진처럼 입력하였을 때 gazebo가 실행되지 않습니다.

**Reply by kimsooyoung** [reply]
로봇 등장에 문제가 있어 보입니다. 수요일 강의 시 다루겠습니다.

**Reply by kimsooyoung** [reply]
수요일 강의 참여하셨다면, gazebo와 로봇 등장을 분리하여 실행시킬 수 있을 것 입니다.
해당 작업 해보시고 어디서 문제가 생겼는지 찾아 다시 알려주세요

**Reply by ohzzang** [reply]
turtlesim  관련 질문은 답해주신 export 방법으로 해결하였고, 수요일 강의 방법으로 gazebo 도 실행 되었습니다.
